spec/controllers/leagues_controller_spec.rb

```diff
@@ -74,7 +74,7 @@
           @league.add_team(team2)
           get :ranking, params: { id: @league.to_param }, session: valid_session
           ranking_entries = controller.instance_variable_get(:@ranking_entries)
-          expect(ranking_entries.first.name).to be < ranking_entries.second.name
+          expect(ranking_entries.first.team.name).to be < ranking_entries.second.team.name
```

Comment:
for how often you had to edit that, a getter for the name doesn't seem like such a bad idea ;)

Reply:
Not that much ;)
And now the work is done.